### PR TITLE
[svelte-kit-scss] Fix styling issues for repository card component

### LINKED ISSUE #1130: [svelte-kit-scss] Fix styling issues for repository card component

### PR DESCRIPTION
**Pull request by ihardz**
resolves #1130

Results: 

image

image

### CODE REVIEW

**Review by iansamz — CHANGES_REQUESTED** [review]
Remove commented code

File: svelte-kit-scss/src/lib/components/RepositoryList/RepositoryCard/RepositoryCard.spec.ts [hunk 1]
```diff
@@ -5,23 +5,25 @@ import { relativeTimeFmt } from '../../../helpers';
...
-    ['forksCount', MOCK_REPOSITORY_CARD_VIEW_MODEL.forksCount],
-    ['license_name', MOCK_REPOSITORY_CARD_VIEW_MODEL.license?.name],
-    ['updated', relativeTimeFmt(MOCK_REPOSITORY_CARD_VIEW_MODEL.updatedAt || '')],
-  ])('should render: %s', (testId, expectedValue) => {
-    const element = screen.getByTestId(testId);
-    const expectedText = String(expectedValue);
-    expect(element.innerHTML).toEqual(expectedText);
+    it.each([
+      ['name', MOCK_REPOSITORY_CARD_VIEW_MODEL.name],
+      // ['language', MOCK_REPOSITORY_CARD_VIEW_MODEL.language],
```
Comment (BigAB) [inline svelte-kit-scss/src/lib/components/RepositoryList/RepositoryCard/RepositoryCard.spec.ts:17]: This comment too

File: svelte-kit-scss/src/lib/components/RepositoryList/RepositoryCard/RepositoryCard.svelte [hunk 2]
```diff
@@ -3,6 +3,7 @@
   import { Law16, Star16, RepoForked16 } from 'svelte-octicons';
   import { LANGUAGE_COLORS } from '$lib/constants/language-colors';
   import type { RepositoryCardViewModel } from '../view-models';
+  import ProgrammingLanguage from '$lib/components/shared/ProgrammingLanguage/ProgrammingLanguage.svelte';
```
Comment (BigAB) [inline svelte-kit-scss/src/lib/components/RepositoryList/RepositoryCard/RepositoryCard.svelte:6]: There is a lint warning in this file above 
> 'LANGUAGE_COLORS' is defined but never used

File: svelte-kit-scss/src/lib/components/RepositoryList/RepositoryCard/RepositoryCard.svelte [hunk 3]
```diff
@@ -16,111 +17,166 @@
...
-  {/if}
-  <div class="stars">
-    <Star16 />
-    <span class="star-count" data-testid="stargazersCount">{stargazersCount}</span>
+    {#if description}
+      <div class="description">
+        <span data-testid="description">{description}</span>
+      </div>
+    {/if}
+    <!-- {#if topics}
```
[the remark this answers is not in the document]
Comment (BigAB) [reply svelte-kit-scss/src/lib/components/RepositoryList/RepositoryCard/RepositoryCard.svelte:37]: That's just spreading the code across 2 PRs.   Just remove the commented code here, put it in a file or a snippet to use in the next PR

**Review by BigAB — CHANGES_REQUESTED** [review]
Remove commented code and the now unnecessary import and this should be good to go

**Review by BigAB — APPROVED** [review]
LGTM

**Pull request merged**